docs/citus.rst

```diff
@@ -0,0 +1,196 @@
+.. _citus:
+
+Citus Support
+=============
+
+The usual ``pg_autoctl`` commands work both with Postgres standalone nodes
+and with Citus nodes.
+
+.. figure:: ./tikz/arch-citus.svg
+   :alt: pg_auto_failover architecture with a Citus formation
+
+   pg_auto_failover architecture with a Citus formation
+
+When using pg_auto_failover with Citus, a pg_auto_failover *formation* is
```

Comment:
Maybe cluster instead of formation?

```suggestion
When using pg_auto_failover with Citus, a pg_auto_failover *cluster* is
```

Reply:
Well, we use `--formation`, document the term in our glossary, and also have `pg_autoctl create formation`, so I guess that's already set... am I missing something here?